docs/src/usage.md

```diff
@@ -28,6 +28,46 @@ Hello world, I am rank 1 of 3
 
 The [`mpiexec`](@ref) function is provided for launching MPI programs from Julia itself.
 
+
```

Comment:
This is the wrong line to attach this comment to, but in line 7 it says "operaions" instead of "operations"